iree/compiler/Dialect/VM/Target/C/CModuleTarget.cpp

```diff
@@ -148,11 +148,88 @@ static LogicalResult initializeGlobals(IREE::VM::ModuleOp moduleOp,
   return success();
 }
 
+static LogicalResult translateBranchOp(IREE::VM::BranchOp branchOp,
```

Comment:
are there any lit tests that could be added for these? it'd help demonstrate what's happening and be good examples of the conversion

Reply:
We have those for the `mlir::BranchOp` and the `mlir::CondBranchOp` (https://github.com/iml130/mlir-emitc/blob/main/test/Target/control_flow.mlir), so we can also add similar lit tests to IREE.